app/controllers/tags_controller.rb

```diff
@@ -0,0 +1,18 @@
+class TagsController < ApplicationController
+  def index
+    @tags = Tag.all
+  end
+
+  def show
+    @tag = Tag.find(params[:id])
+    @questions = Question.all.select{|question| question.tags.include? @tag}
```

Comment:
This is remarkably inefficient. It's loading ALL questions, then finding questions that are associated with a particular tag. Instead you could define `has_many :questions` on `Tag`. Then use it here, `@questions = @tag.questions`

```